lattice-control/rust/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "wasmcloud-interface-lattice-control"
-version = "0.16.1"
+version = "0.17.0"
```

Comment:
adding an optional field should be a patch bump

Reply:
it's still a public API change, though.